web/libs/editor/src/components/BottomBar/buttons.tsx

```diff
@@ -94,49 +92,49 @@ type SkipButtonProps = {
 export const SkipButton = memo(
   observer(({ disabled, store, onSkipWithComment }: SkipButtonProps) => {
     return (
-      <ButtonTooltip key="skip" title="Cancel (skip) task: [ Ctrl+Space ]">
-        <Button
-          aria-label="skip-task"
-          disabled={disabled}
-          onClick={async (e) => {
-            const action = () => store.skipTask({});
-            const selected = store.annotationStore?.selected;
+      <Button
+        aria-label="skip-task"
```

Comment:
These buttons are used in a list, so should have a `key` here to not trigger react warnings
```suggestion
      <Button
        key="skip"
        aria-label="skip-task"
```